docs/pages/debugging/devtools-plugins.mdx

```diff
@@ -0,0 +1,201 @@
+
+## Adding a dev tools plugin to your project
+
+Dev tools plugins are installed as packages in your project. A plugin will typically have a small amount of connecting code that you add to your app in order to enable two-way communication between your app and the plugin. This code will generally be invoked in your app's root component, especially if the plugin is inspecting a module that is available for the entire time your app is running. It is recommended that you wrap this code in a conditional that only runs in development mode, so that it is not included in production builds of your app:
+
+```jsx
+if (__DEV__) {
+  // initialize your DevTools plugin here
+}
+```
+
+In some cases, you may need to interact with a plugin directly. All plugins communicate through exports from `expo/devtools`, and you can send and listen to messages through `connectPluginFromDevToolsAsync`:
+
+```jsx
+import { connectPluginFromDevToolsAsync } from 'expo/devtools';
+
+const client = await connectPluginFromDevToolsAsync();
+
+client.sendMessage('ping', { from: 'app' });
+client.addMessageListener('pong', (data) => {
+  console.log('receiving pong', data);
+});
+```
+
+### Compatibility with Expo Go and Development Builds
+
+Dev tools plugins themselves should only include JavaScript code, meaning they are generally compatible with both Expo Go and [Development Builds](development-builds/introduction/), and it should not require creating a new development build in order to add a dev tools plugin. If the underlying module that the plugin is inspecting is from a package that includes native code that is not already part of Expo Go, then you would need to create a new development build with that package included in order to use both the component and the plugin.
+
+For example, if you had a dev tools plugin that inspected [React Native Firebase](), that would not work with Expo Go, as React Native Firebase includes native code that is not already part of Expo Go. You would need to create a development build in order to use React Native Firebase, and then you could also use the dev tools plugin.
+
+## Using a dev tools plugin
+
+Once your dev tools plugin is installed and any required connecting code is added to your project. Start your dev server up with `npx expo start`, and then press <kbd>shift</kbd> + <kbd>m</kbd> to open the list of dev tools plugins. Select the plugin you want to use, and it will open in a new Chrome window.
+
+> When starting the a dev server with the Expo CLI, there is an option to press <kbd>?</kbd> to "show all commands". This will show additional commands, including the shortcut to open "more tools." Dev tools plugins can also be selected in this menu.
```

Comment:
```suggestion
> When starting the dev server with the Expo CLI, there is an option to press <kbd>?</kbd> to **show all commands**. This shows additional commands, including the shortcut to open **more tools**. Dev tools plugins can also be selected in this menu.
```

- Incorrect use of articles, let's stick to using one
- Let's use bold text to highlight something from our tools/settings instead of quotes for consistency